AnkiDroid/src/main/java/com/ichi2/async/Connection.java

```diff
@@ -77,6 +77,16 @@
      * the process will be paused and the sync can fail due to timing conflicts with AnkiWeb.
      */
     private final PowerManager.WakeLock mWakeLock;
+    private long kBytesReceived, kBytesSent;
+    private long filesDownloaded=0;
```

Comment:
https://github.com/ankidroid/Anki-Android/wiki/Code-style

```suggestion
    private long filesDownloaded = 0;
```

Reply:
Done